modules/builders-virtual-environment.adoc

```diff
@@ -104,7 +104,7 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IldfQUJkaDVmb3ltTHZ0dGZMYjhIWnYxZTQzN2dJVEJxcDJscldS
 +
 [source,terminal]
 ----
-$ oc get route -n quay-enterprise
+$ oc get route -n quay-enterprise —duration
```

Comment:
There is no such option for `oc get route`. Please remove.

Reply:
Remove this option , as Yussuf has suggested

Reply:
Done. Thanks.